nf_core/pipeline-template/{{cookiecutter.name_noslash}}/environment.yml

```diff
@@ -6,6 +6,7 @@ channels:
   - bioconda
   - defaults
 dependencies:
+  - python=3.7
```

Comment:
```suggestion
  - conda-forge::python=3.7
```

Reply:
Actually needs to be 3.7.4 I think for the linting to pass, as it must match an exact release number.